packages/cli/src/commands/equip.ts

```
@@ -0,0 +1,249 @@
+    if (selection.configs.includes('@outfitter/husky-config')) {
+      const gitHooks = await confirm({
+        message: 'Initialize git hooks?',
+        default: true,
+      });
+      
+      if (gitHooks) {
+        const hooksSpinner = ora('Setting up git hooks...').start();
+        try {
+          // TODO: Initialize husky
+          hooksSpinner.succeed('Git hooks initialized');
+        } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)